addons/web/static/src/js/core/popover.js

```diff
@@ -90,34 +128,69 @@ odoo.define('web.Popover', function () {
             }
         }
 
-        //--------------------------------------------------------------------------
-        // Private
-        //--------------------------------------------------------------------------
+        //----------------------------------------------------------------------
+        // Handlers
+        //----------------------------------------------------------------------
 
         /**
+         * Toggles the popover depending on its current state.
+         *
+         * @param {MouseEvent} ev
+         */
+        _onClick(ev) {
```

Comment:
missing `@private` tag (some other handler functions too)

Reply:
Done, those were method that were public before, forgot to add it.